pom.xml

```diff
@@ -367,54 +367,58 @@
                 <version>${maven-license-plugin.version}</version>
                 <inherited>true</inherited> <!-- for multi-module config -->
                 <configuration>
-                    <header>${license.header}</header>
                     <aggregate>true</aggregate> <!-- for multi-module config -->
                     <skipExistingHeaders>false</skipExistingHeaders>
-                    <includes>
-                        <!-- Add files where to include licensing -->
-                        <include>src/**/*.java</include>
-                        <include>src/**/*.xml</include>
-                        <include>src/**/*.proto</include>
-                        <include>**/*.css</include>
-                        <include>**/*.sh</include>
-                    </includes>
-                    <excludes>
-                        <!-- Add files to exclude from licensing -->
+                    <!-- Add comment style for protocol buffers -->
+                    <mapping>
+                        <java>SLASHSTAR_STYLE</java>
+                        <proto>DOUBLESLASH_STYLE</proto>
```

Comment:
Just replace the ASL Header in the proto file with the standard Slashstar format one ?
That's what HBase and Phoenix uses.